python/nano/src/bigdl/nano/deps/openvino/openvino_api.py

```diff
@@ -27,3 +27,8 @@ def bind_openvino_methods(pl_model):
 def export(model, input_sample=None, xml_path="model.xml"):
     from . import torch_funcs
     torch_funcs.export(model, input_sample, xml_path)
```

Comment:
should we remove this `export` and `bind_openvino_methods`? Since the usage of `Trainer.compile(openvino=True)` is removed anyway.